llvm/lib/Target/SystemZ/SystemZFrameLowering.h

```diff
@@ -97,6 +106,11 @@ class SystemZELFFrameLowering : public SystemZFrameLowering {
     return usePackedStack(MF) ? SystemZMC::ELFCallFrameSize - 8 : 0;
   }
 
+  // Return the offset of the return address.
+  virtual int getReturnAddressOffset(MachineFunction &MF) const override {
```

Comment:
We don't normally use `virtual` on the function definitions, only on the declarations.

Reply:
Changed.